packages/@webex/internal-plugin-ediscovery/test/unit/spec/transforms.js

```diff
@@ -289,16 +289,17 @@ describe('EDiscovery Transform Tests', () => {
       });
 
       it('Decrypt function throws an exception', () => {
+        const expectedError = new Error('Failed to acquire a key');
         ctx.webex.internal.encryption.decryptText = sinon
           .stub()
-          .returns(Promise.reject(new Error('@@@@@@')));
+          .returns(Promise.reject(expectedError));
 
         const result = Transforms.decryptReportContent(ctx, object, reportId)
           .then(() => {
             assert.fail('Decrypt did not fail as expected');
           })
           .catch(() => {
-            assert.isDefined(activity.error);
+            assert.deepEqual(activity.error, expectedError);
```

Comment:
Nothing to do with my PR, but Adding a more robust assert here for the desired exception, rather than any exception.